ext/pcre/php_pcre.c

```diff
@@ -705,13 +707,11 @@ PHPAPI pcre_cache_entry* pcre_get_compiled_regex_cache_ex(zend_string *regex, in
 		}
 	}
 
-	if (*pp == 0) {
+	if (pp == end_p) {
```

Comment:
Maybe for consistency with other changes:
```suggestion
	if (pp >= end_p) {
```

Reply:
Applied this in the latest commit.